bin/public/installNode.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+'''
+Install NodeJS server
+
+'''
+
+__author__ = "rikard.stenlund@fareoffice.com"
+__copyright__ = "Copyright 2014, The System Console project"
+__maintainer__ = "Rikard Stenlund"
+__email__ = "syco@cybercow.se"
+__credits__ = ["Daniel Lindh"]
+__license__ = "???"
+__version__ = "1.0.0"
+__status__ = "Production"
+
+import general
+from general import x
+from scopen import scOpen
+import app
+import version
+import os
+import iptables
+
+# The version of this module, used to prevent the same script version to be
+# executed more then once on the same host.
+SCRIPT_VERSION = 1
+
+
+def build_commands(commands):
+	commands.add("install-node", install_nodejs, help="Install nodejs on server (version tested: 0.12.2)")

```

Comment:
Please change command name to be less generic, like maybe install-node-js, install-node could mean a lot of things.